lambda/ecr_scan_notifications.tf

```diff
@@ -48,3 +48,12 @@ resource "aws_iam_role_policy_attachment" "ecr_scan_notifications_lambda_role_po
   policy_arn = aws_iam_policy.ecr_scan_notifications_lambda_policy.*.arn[0]
   role       = aws_iam_role.ecr_scan_notifications_lambda_iam_role.*.name[0]
 }
+
+resource "aws_lambda_permission" "lambda_permissions" {
```

Comment:
Has  `terraform fmt` been run on this file?

Reply:
No, sorry.